### ISSUE TITLE
How are the settings files generated / handled for hardware acceleration configurations?

### ISSUE THREAD

**Issue by paxdriver** [OpenShot/openshot-qt#4611]
I'm running Linux appImage of version 2.5.1 and have nvidia gpu working on it with accelerated encoding. The new updated version won't allow me to select hardware encoding with version 2.6.1 as I've been doing regularly on version 2.5.1. 

vdpau and vainfo return successfully and I can confirm that I'm running the latest proprietary driver (driver 470, rendering weekly videos for 1 yr, and I even tried driver 495 trying to get this latest version of OpenShot to work with it, but with no luck). 

I've done a complete removal and reinstall of my graphics drivers. I've tried setting global environment variable by entering 'LIBVA_DRIVER_NAME=nvidia' in the terminal and then launching the appImage but that doesn't work either.

I've also had this problem on the appImage v.2.5.1 from time to time too, but deleting the .qt-openshot folder was enough to get v.2.5.1 to allow me to select hardware acceleration again. I then noticed that it recreates the settings file inside of the qt-openshot folder so I saved a working copy of it, and now when in v2.5 the hardware acceleration switches off I just paste my saved settings file over it once and no issues next time it launches.

BUT! Version 2.6.1 won't allow me to select my graphics card and the trick I've been using overwriting the settings file no longer works... 

**Question 1) Can you help me figure out what's going on with the hardware acceleration being greyed out and not available to the newest version of OpenShot please?

Question 2) Can you tell me where / why / what is the appImage doing differently when it launches compared to v.2.5.1 so I can try to find a hacky work around?**

Screenshot is v.2.5.1 compared to v.2.6.1 and I tried today's daily build as well but got the same results as 2.6.1. Older version is on the left, the latest version is on the right. Below I've copied the results from running 'vainfo' in case it helps.

I love your product and promote it proudly on my podcast, thank you for the hard work I hope you can help me get the hardware acceleration working on my system!

libva info: VA-API version 1.1.0
libva info: va_getDriverName() returns 0
libva info: Trying to open /usr/lib/x86_64-linux-gnu/dri/nvidia_drv_video.so
libva info: Found init function __vaDriverInit_1_0
libva info: va_openDriver() returns 0
vainfo: VA-API version: 1.1 (libva 2.1.0)
vainfo: Driver version: Splitted-Desktop Systems VDPAU backend for VA-API - 0.7.4
vainfo: Supported profile and entrypoints
      VAProfileMPEG2Simple            :	VAEntrypointVLD
      VAProfileMPEG2Main              :	VAEntrypointVLD
      VAProfileMPEG4Simple            :	VAEntrypointVLD
      VAProfileMPEG4AdvancedSimple    :	VAEntrypointVLD
      <unknown profile>               :	VAEntrypointVLD
      VAProfileH264Main               :	VAEntrypointVLD
      VAProfileH264High               :	VAEntrypointVLD
      VAProfileVC1Simple              :	VAEntrypointVLD
      VAProfileVC1Main                :	VAEntrypointVLD
      VAProfileVC1Advanced            :	VAEntrypointVLD

![openshotScreenshots](https://user-images.githubusercontent.com/49280957/146950235-739b6f29-535c-402b-b733-99aa9e9d12a4.png)

**Comment by daetsch** [comment]
Just switched to 2.6.1. Same here.

**Comment by paxdriver** [comment]
> Just switched to 2.6.1. Same here.

I ran the new version in another distro of linux (either popOS or manjaro, can't remember which I got it working in) but the renders were using like 25% of thread capacity, it was leaving entire threads unused (only 12 of 16 active at any given time during the render) and the gpu encoding was slower too. 

Ultimately, I went from getting consistent 80-88fps rendering 2hr podcast audio/video at 60fps on medium at 320kbps on Linux Mint on v.2.5.1 to getting less than 50fps on the exact same file and hardware on the latest version of OpenShot. I was hoping for render and scrub performance improvements but there doesn't seem to be any beneit to the newer version other than built-in stickers being easily available lol. 

Hopefully the devs respond but until then I'll wait til I get a 15% or better performance boost before mucking around with all this. It was hard enough trying to figure out the workaround to get 2.5 running in the first place lol
